com.openwall.John.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "com.openwall.John",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "john",
+    "tags": [ "1.9J1-4bf9652", "1.9.0 Jumbo 1" ],
+    "finish-args": [
+        "--filesystem=home"
+    ],
+    "cleanup" : [
+        "/include",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "libpcap",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.tcpdump.org/release/libpcap-1.8.1.tar.gz",
+                    "sha256": "673dbc69fdc3f5a86fb5759ab19899039a8e5e6c631749e48dcd9c6f0c83541e"
+                }
+            ]
+        },
+        {
+            "name": "librexgen",
+            "buildsystem": "cmake",
+            "subdir": "./src",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/teeshop/rexgen/archive/1.4.0.zip",
+                    "sha256": "a7903c21e20e0817b7c760bd13fcff802839b71c1f95db926cf52bb93f13f69d"
+                }
+            ]
+        },
+        {
+            "name": "john",
+            "buildsystem": "simple",
+            "subdir": "src",
+            "build-commands": [
+                "###### build script ######",
+                "../build.sh"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/claudioandre-br/JohnTheRipper/archive/1.9.0-jumbo-1.zip",
+                    "sha256": "56635f14467a18d8835c6d586ec108bee4b0bad437341f325a817536623956e3"
+                },
+                {
+                    "type": "patch",
+                    "path": "confined-system-wide-build.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "use-regex-1.4.patch"
+                },
+                {
+                    "type": "file",
+                    "path": "build.sh",
+                    "dest-filename": "build.sh"
+                },
+                {
+                    "type": "file",
+                    "path": "com.openwall.John.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "com.openwall.John.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "com.openwall.John.png"
+                }
+            ],
+            "post-install": [
+                "###### Install JtR system wide ######",
+                "cp -r ../run/.   /app/bin",
+                "cp -r ../doc/    /app/",
+                "cp    ../README* /app/",
+                "###### Bash Completion ######",
```

Comment:
bash completion is not exported. Not sure if it's useful

Reply:
The examples I found suggest that copy to the _magic place_ will work. Do you have any example that I can follow?